docs/compile_guide.md

```diff
@@ -69,24 +75,48 @@ You can compile the code by one of the three approaches:
       $ cd $GOPATH/src/github.com/vmware/harbor
       $ make install
    ```
-   
+
+*  Build, install and run Harbor with Notary:
```

Comment:
Update title in the last section:
 Build, install and bring up Harbor without Notary: